torch/_dynamo/variables/torch.py

```diff
@@ -128,9 +128,14 @@ class TorchVariable(VariableTracker):
 
     def __init__(self, value, **kwargs):
         super().__init__(**kwargs)
+        try:
+            if value in tensor_dunder_fns_remap:
+                value = tensor_dunder_fns_remap[value]
+        except TypeError as e:
+            # is it cleaner to somehow exclude unhashable types from the `in` check above?
+            if "unhashable type" not in e.args[0]:
```

Comment:
Don't do this, parsing strings from exceptions is weird. Use https://docs.python.org/3/library/collections.abc.html#collections.abc.Hashable

Reply:
i think i actually want to insert an `inspect.isfunction(value)` before checking the remap.  is that a better way to do this?